### ISSUE TITLE
Request: Bring back Ctrl+^

### ISSUE THREAD

**Issue by timothycosta** [johnno1962/InjectionIII#56]
Any chance of bringing back the original Ctrl+^ behavior instead of only using the file watcher?

I find the file watcher results in my app crashing in the simulator every time I touch a storyboard, change something in git, or build for my device.  Not to mention that I sometimes save while thinking, not only when I'm ready to see my changes.

Building everything is fine (if memory serves, originally it only worked within the same file?), but I would like to do it at a moment of my own choosing.

I'm using the Xcode 10 version.

**Comment by johnno1962** [comment]
Hi Timothy, sorry, no can do since the app became a stand alone app. You can turn the file watcher off and on using the menu on the icon if you’re injecting when you don’t want to be.

**Comment by timothycosta** [comment]
Enabling and disabling file watching wouldn't really help as I don't want to pay attention to it.

I took a look at `InjectionServer.mm` and the app delegate.  What I was thinking was that the file watcher could just keep an NSMutableSet of files that have changed since the last injection, instead of immediately calling `inject(swiftSource)`.  Then add the hotkey in `AppDelegate.mm` to trigger compile of the files in that list.

The only obvious issue I see with that is emptying the set when the app is re-run. 

I wanted to try this out myself, but when I build from source Injection doesn't seem to work.  It crashes every time I choose "Start Injection".  I've edited my app delegate to point to he locally built version.  

I'm using Xcode 10 and the version in source doesn't seem to match up with that either.  Any help there?
